src/views/partials/adminSidebar.hbs

```diff
@@ -1,33 +1,33 @@
 <aside class="sidebar">
     <div class="admin-tabs">
-        <div class="profile">
-            <i class="far fa-user-circle"></i>
+        <div class="profile tab focus">
```

Comment:
@ahmedisam99 These could all be `<a>` links, so then you wouldn't need the javascript to add click handlers.  You'd then just need a few style rules to get them to look like the divs that you currently have (display: block and maybe some color / text-decoration changes).  I like to try and use `<a>` tags for navigation whenever possible - then the user can right-click them and open in a new tab, for example.